cypress/locators/radioButton/index.js

```diff
@@ -1,17 +1,13 @@
 import {
-  RADIO_BUTTON,
   RADIO_BUTTON_COMPONENT,
   RADIO_BUTTON_GROUP_COMPONENT,
   RADIO_BUTTON_FIELDSET_GROUP,
 } from './locators';
 
-// component preview locators
-export const radioButtonComponent = () => cy.iFrame(RADIO_BUTTON_COMPONENT);
-export const radioButtonByPosition = position => cy.iFrame(RADIO_BUTTON).eq(position);
-
 // no Iframe locators
```

Comment:
do we need this comment if we are using one type of locators ?

Reply:
normally we have `// component preview locators` so I replaced it